tests/test_deblend.py

```diff
@@ -234,6 +256,61 @@ def test_deblend_task(self):
         skipped = largeFootprint | denseFootprint
         np.testing.assert_array_equal(skipped, catalog["deblend_skipped"])
 
+    def test_continuity(self):
+        """This test ensures that lsst.scarlet.lite gives roughly the same
+        result as scarlet.lite
+
+        This test can be removed once the deprecated scarlet.lite
+        module is removed from the science pipelines.
+        """
+        oldCatalog, oldModelData, oldConfig = self._deblend("old_lite")
+        catalog, modelData, config = self._deblend("lite")
+
+        # Ensure that the deblender used different versions
+        self.assertEqual(oldConfig.version, "old_lite")
+        self.assertEqual(config.version, "lite")
+
+        # Check that the PSF and other properties are the same
+        assert_almost_equal(oldModelData.psf, modelData.psf)
+        self.assertTupleEqual(oldModelData.bands, modelData.bands)
+        self.assertTupleEqual(tuple(oldModelData.blends.keys()), tuple(modelData.blends.keys()))
+
+        # Make sure that the sources have the same IDs
+        for i in range(len(catalog)):
+            self.assertEqual(catalog[i]["id"], oldCatalog[i]["id"])
+
+        for blendId in modelData.blends.keys():
+            oldBlendData = oldModelData.blends[blendId]
+            blendData = modelData.blends[blendId]
+
+            # Check that blend properties are the same
+            self.assertTupleEqual(oldBlendData.origin, blendData.origin)
+            self.assertTupleEqual(oldBlendData.shape, blendData.shape)
+            self.assertTupleEqual(oldBlendData.psf_center, blendData.psf_center)
+            self.assertTupleEqual(tuple(oldBlendData.sources.keys()), tuple(blendData.sources.keys()))
+            assert_almost_equal(oldBlendData.psf, blendData.psf)
```

Comment:
This is the PSF per blend, which is different from the `oldModelData.psf` above, right?

Reply:
Yes. `oldModelData.psf` and `modelData.psf` is the model PSF that all of the blends are generated in.